lib/yabeda/sidekiq.rb

```diff
@@ -26,6 +26,7 @@ def self.config
       group :sidekiq
 
       counter :jobs_enqueued_total, tags: %i[queue worker], comment: "A counter of the total number of jobs sidekiq enqueued."
+      counter :jobs_rerouted_total, tags: %i[queue worker], comment: "A counter of the total number of rerouted jobs sidekiq enqueued."
```

Comment:
WDYT, will it be useful to track both original and resulted queues in this metric?

Like this:
```suggestion
      counter :jobs_rerouted_total, tags: %i[from_queue to_queue worker], comment: "A counter of the total number of rerouted jobs sidekiq enqueued."
```